src/OpenLoco/src/Windows/Construction/OverheadTab.cpp

```diff
@@ -505,6 +503,8 @@ namespace OpenLoco::Ui::Windows::Construction::Overhead
                 clipped->x += screenPos.x;
                 clipped->y += screenPos.y;
```

Comment:
Can you move those lines to 495, its not really used by anything other than pushRenderTarget.

Reply:
No, this is the first line with all the state

Reply:
Hm right